.github/workflows/pull-request.yml

```diff
@@ -66,6 +67,30 @@ jobs:
         working-directory: packages/transaction_signing_gateway
         if: env.changes_transaction_signing_gateway == 'true'
         run: flutter test
+        
+      ###########################################################################################
+      ## COSMOS_NODE_CLIENT
+      ###########################################################################################
+
+      - name: cosmos_node_client - pub get
+        working-directory: packages/cosmos_node_client
+        if: env.changes_transaction_signing_gateway == 'true'
+        run: flutter pub get
+
+      - name: cosmos_node_client - check codestyle
+        working-directory: packages/cosmos_node_client
+        if: env.cosmos_node_client == 'true'
+        run: find lib test -name "*.dart" -not -name "*.g.dart" -not -name "*.freezed.dart" -not -name "*.gen.dart" -exec dart format -l 120 --set-exit-if-changed {} +;
+
+      - name: cosmos_node_client - dart analysis
+        working-directory: packages/cosmos_node_client
+        if: env.cosmos_node_client == 'true'
+        run: flutter analyze
+
+     # - name: cosmos_node_client - tests
+        #working-directory: packages/cosmos_node_client
+        #if: env.cosmos_node_client == 'true'
+        #run: flutter test
```

Comment:
any reason this is commented out?

Reply:
We don't have any tears right now and the ci fails because of that

Reply:
lets add one, even 2+2=4 test just to enable the CI from the beginning. you can always remove the test as soon as you add proper ones later on

Reply:
Okay